src/pybind/rados/c_rados.pxd

```diff
@@ -191,6 +191,13 @@ cdef extern from "rados/librados.h" nogil:
 
     int rados_nobjects_list_open(rados_ioctx_t io, rados_list_ctx_t *ctx)
     int rados_nobjects_list_next(rados_list_ctx_t ctx, const char **entry, const char **key, const char **nspace)
+    int rados_nobjects_list_next2(rados_list_ctx_t ctx,
```

Comment:
could you remove `rados_nobjects_list_next(...)` from the .pxd and .pxi files respectively?